src/dpp/discordvoiceclient.cpp

```diff
@@ -1166,20 +1166,35 @@ discord_voice_client& discord_voice_client::set_send_audio_type(send_audio_type_
 
 discord_voice_client& discord_voice_client::send_audio_raw(uint16_t* audio_data, const size_t length)  {
 #if HAVE_VOICE
-	const size_t max_frame_bytes = 11520;
+	static const size_t max_frame_bytes = SEND_AUDIO_RAW_MAX_LENGTH;
```

Comment:
Why the addition of `static` here? I'm guessing as an "optimization"? You're going to get a cache miss which is much more costly than the initialization of a size_t in the register.
Also, if we're going to edit this line, can we make it constexpr?